woocommerce/sensei-content-drip
# Use WordPress timezone when dripping lessons

Fixes #196

### Changes proposed in this Pull Request

* Bumps WP minimum version to 5.3. This is in line with Sensei LMS and also saves lots of annoying compatibility checks for WP 5.3's new date functions.
* Standardize all date time usage so that content will drip using WordPress' timezone. 
* Reorganizes tests to conform with Sensei test structure.

### Testing instructions

* Using various timezones relative to UTC/PHP timezone, check to make sure both relative/dynamic drips and absolute drips happen at the correct date.
